### PR TITLE
ci: migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by MaximDevoir**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
     name: release
     runs-on: ubuntu-latest
+    needs: test
+    if: |
+      github.event_name == 'push'
+      && github.ref == 'refs/heads/master'
+      && github.repository == 'probot/create-probot-app'
     steps:
       - uses: actions/checkout@master
       - uses: actions/setup-node@v1
```
Comment (gr2m) [inline .github/workflows/release.yml]: Thanks for getting this going Maxim! 

I would suggest we create a separate `.github/workflows/test.yml` file for the tests, similar to what I did in most of the Octokit repositories: https://github.com/octokit/core.js/tree/master/.github/workflows.

I don't think we need to run tests before releasing from master, the PR workflow is already enforced, so tests are guaranteed to be green on master `push` events.

What do you think?

Comment (MaximDevoir) [reply .github/workflows/release.yml]: That will work.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - "greenkeeper/**"
```
Comment (MaximDevoir) [inline .github/workflows/test.yml]: In reference to [other conversation](https://github.com/probot/create-probot-app/pull/112/files#r368074993), the `test` workflow from octokit/core.js includes the `master` branch. Here, I've removed it as changes merged to master are guaranteed to be successful.

Comment (gr2m) [reply .github/workflows/test.yml]: good point

**Pull request merged**